src/Base/Placement.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "PreCompiled.h"
 
+#include "Console.h"

```

Comment:
Looks like you should rather `#include <fmt/format.h>` to avoid to pull in unneeded stuff.